sdk/tests/client/error.rs

```diff
@@ -38,10 +35,4 @@ fn stringified_error() {
         &serde_json::to_string(&error).unwrap(),
         "{\"type\":\"inputSelection\",\"error\":\"insufficient amount: found 0, required 100\"}"
     );
-
-    let error = Error::InputSelection(IsaError::Block(BlockError::InvalidAddress));
```

Comment:
We should replace this with a similarly formatted error, this test case if for stringified errors of various shapes